packages/fiber/src/core/hooks.ts

```diff
@@ -80,23 +80,19 @@ export function useLoader<T, U extends string | string[]>(
 ): U extends any[] ? BranchingReturn<T, GLTF, GLTF & ObjectMap>[] : BranchingReturn<T, GLTF, GLTF & ObjectMap> {
   // Use suspense to load async assets
   const keys = (Array.isArray(input) ? input : [input]) as string[]
-  const results = useAsset(loadingFn<T>(extensions, onProgress), Proto, ...keys)
+  const results = suspend(loadingFn<T>(extensions, onProgress), [Proto, ...keys], { equal: is.equ })
   // Return the object/s
   return (Array.isArray(input) ? results : results[0]) as U extends any[]
     ? BranchingReturn<T, GLTF, GLTF & ObjectMap>[]
     : BranchingReturn<T, GLTF, GLTF & ObjectMap>
 }
 
-useLoader.preload = function <T, U extends string | string[]>(
-  Proto: new () => LoaderResult<T>,
-  input: U,
-  extensions?: Extensions,
-) {
+useLoader.preload = function <T, U extends string | string[]>(Proto: new () => LoaderResult<T>, input: U) {
   const keys = (Array.isArray(input) ? input : [input]) as string[]
-  return useAsset.preload(loadingFn<T>(extensions), Proto, ...keys)
+  return preload(Proto, ...keys)
```

Comment:
TS may need some work. The use of `...keys` is getting me:
> A spread argument must either have a tuple type or be passed to a rest parameter. ts(2556)

Reply:
... keys as Tuple<undefined> or something like this should be ok